src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -983,6 +983,48 @@ public bool EnableSelection
             }
         }
 
+        /// <summary>
+        /// The start index for selection.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done, Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int SelectedTextStart
+        {
+            get
+            {
+                int temp;
+                GetProperty(TextEditor.Property.SELECTED_TEXT_START).Get(out temp);
+                return temp;
+            }
+            set
+            {
+                SetProperty(TextEditor.Property.SELECTED_TEXT_START, new PropertyValue(value));
+                NotifyPropertyChanged();
+            }
+        }
+
+        /// <summary>
+        /// The end index for selection.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done, Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int SelectedTextEnd
+        {
+            get
+            {
+                int temp;
+                GetProperty(TextField.Property.SELECTED_TEXT_END).Get(out temp);
```

Comment:
I think it is correct to modify TextField to TextEditor//

Reply:
Hello, @Ali-Alzyoud , I think this is not resolved yet. Could you check it again?